samples/MonoSanityClient/Examples.cs

```diff
@@ -33,7 +33,7 @@ public static void TriggerException(string message)
 
         public static void InvokeWipedMethod()
         {
-            new HttpClientHandler();
+            new HttpClientHandler().Dispose();
```

Comment:
Should we then have another test that checks that calling the `ctor` doesn't throw?

Reply:
This test isn't trying to specify which parts of the BCL are wiped; it's just trying to test that when something *is* wiped, the exception it reports is the one we expect.